### PR TITLE
Create disclosures.md

### PR DESCRIPTION
**Pull request by carlinhocr**
### Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

**Review by marekolszewski — COMMENTED** [review]
Should we also point people to our HackerOne profile?

File: disclosures.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Vulnerability Disclosure Policy
+
+## Introduction
+
+We at Celo are committed to ensuring the security of the platfomr by protecting their information. This policy is intended to give security researchers
```
Comment (asaj) [inline disclosures.md:5]: typo

**Review by asaj — COMMENTED** [review]
What is the role of this file wrt `SECURITY.md`?

**Comment by asaj** [comment]
There is a `SECURITY.md` for celo-blockchain as well

**Pull request closed without merging**